requirements/requirements-dev.txt

```
@@ -3,6 +3,7 @@ kornia-rs==0.0.8
 mypy[reports]
 numpy
 onnx
+onnxruntime-gpu>=1.16
```

note sure if this is the right marker https://peps.python.org/pep-0508/#environment-markers
```suggestion
onnxruntime-gpu>=1.16  # sys_platform != darwin
```